trl/trainer/training_args.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# coding=utf-8
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from dataclasses import dataclass, field
+from typing import Optional
+
+from transformers import TrainingArguments
+
+
+@dataclass
+class RewardTrainingArguments(TrainingArguments):
```

Comment:
Another name for this could be `RewardConfig` to be more aligned with `PPOConfig`

Reply:
I opted for `RewardConfig` in https://github.com/huggingface/trl/pull/726/commits/e6e346d321654de7a689d44a05ef51d9e2005a34

I've also placed this in a `training_configs.py` module - let me know if you'd prefer a dedicated module per config (currently `PPOConfig` lives in `ppo_config.py`)